post_installation_configuration/storage-configuration.adoc

```diff
@@ -0,0 +1,101 @@
+:context: post-install-storage-configuration
+[id="post-install-storage-configuration"]
+= Post-installation storage configuration
+include::modules/common-attributes.adoc[]
+:gluster: GlusterFS
+:gluster-native: Containerized GlusterFS
+:gluster-external: External GlusterFS
+:gluster-install-link: https://docs.gluster.org/en/latest/Install-Guide/Overview/
+:gluster-admin-link: https://docs.gluster.org/en/latest/Administrator%20Guide/overview/
+:gluster-role-link: https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_storage_glusterfs
+ifdef::openshift-enterprise,openshift-webscale[]
+:gluster: Red Hat Gluster Storage
+:gluster-native: converged mode
+:gluster-external: independent mode
+:gluster-install-link: https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.3/html/installation_guide/
+:gluster-admin-link: https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.3/html/administration_guide/
+:cns-link: https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.3/html/container-native_storage_for_openshift_container_platform/
+endif::[]
+toc::[]
+
+[%interactive]
+* [ ] If using, deploy Red Hat OpenShift Container Storage.
+* [ ] Deploy and configure additional dynamic storage provisioners.
+* [ ] Some storage vendors publish and support their dynamic provisioner separately. Work with your storage team and vendor to determine if they have a CSI provisioner that works with {product-title}.
+* [ ] Several partners have created Operators and certified their storage offerings for {product-title}, which can be found in link:https://marketplace.redhat.com/en-us[Red Hat Marketplace].
```

Comment:
Looks like this item and the last item talk the same thing(csi),  and I did not find the section for this topic(csi).

Reply:
And looks like the index is not organized as the interactive part, don't know if it's ok.